pkg/uploadproxy/uploadproxy_test.go

```diff
@@ -31,6 +35,11 @@ type validateSuccess struct{}
 
 type validateFailure struct{}
 
+func TestUploadProxy(t *testing.T) {
```

Comment:
In general we try to have a single 'suite' file that has the 'Testing.T' in it. We name it: _suite_test.go with the package here being apiserver. It also allows us to set things like logger in the before suite. A decent example is in pkg/controller/controller_suite_test.go